app/components/Tooltip/index.js

```diff
@@ -10,20 +10,46 @@ type Props = {
   className?: string,
   onClick?: () => void,
   style?: Object,
-  list: boolean
+  list?: boolean,
+  renderToThe?: string,
+  pointerToThe?: string
 };
 
 type State = {
-  hovered: boolean
+  hovered: boolean,
+  childrenContainerWidth: number
 };
 
+/**
+ * A tooltip that appears when you hover over the component placed within.
+ * The tooltip will by default be centered, however it supports a 'renderToThe'
+ * prop that will make it render either to the left. The pointer
+ * (the small arrow that points towards the component within the tooltip) will
+ * also default to center, and it can be adjusted with the 'pointerToThe' prop.
+ * Both props can be set as either 'left' or 'right'.
```

Comment:
How do the different combinations look? If you renderToThe left and pointerToThe right, what happens?

Reply:
![Screenshot from 2019-03-28 15-42-52](https://user-images.githubusercontent.com/7977650/55167020-7b664e00-5170-11e9-84b7-b8e38a2a705b.png)
Here is an example of that. The renderToThe prop selects which way the component will render compared to the pointer. The pointerToThe prop selects where the pointer will be on the component within the tooltip. So in this example the pointer is to the right on label within the component, and from there the tooltip renders to the left. If you were to set pointerToThe left the tooltip would render to the left starting at the left end of the label.

Reply:
So "renderToThe" is relative to the pointer, and the pointer is relative to the underlying element? Maybe add a small description about that.

I would also rename the props, maybe something like `pointerPosition` and `tooltipPosition`, or whatever makes sense.

Reply:
Explained that the tooltip is adjusted relative to the pointer, and renamed the props to renderDirection and pointerPosition. Do these names work?

Reply:
Perfect ✅